### ISSUE TITLE
template="pages" appears blank but template="table" works fine

### ISSUE THREAD

**Issue by slochower** [cbouy/mols2grid#8]
Thanks for putting this together! It's really nice. I'm unclear how to proceed to debug this, but I'm able to get images only when I use `template="table"`. If I try `template="pages"` (or leave it blank), then I can only see the "Sort by" bar, screenshot below. I've tested this using `mols2grid` version 0.0.3, in a relatively clean `conda` environment, on macOS using Chrome Version 89.0.4389.90 and Firefox 87.0b9.

![image](https://user-images.githubusercontent.com/5393995/113229386-bd3c9a00-924b-11eb-82f8-25255c3dcaba.png)

**Comment by slochower** [comment]
I should add that I'm calling `mols2grid.display` using a `pandas` dataframe and passing the SMILES column as an argument:

```python
mols2grid.display(
    df,
    fixedBondLength=25,
    smiles_col="SMILES",
    coordGen=True,
    template="table",
)
```

**Comment by cbouy** [comment]
Hmm that's unfortunate! Can you check on Chrome's console if their are any errors?

**Comment by fredrikw** [comment]
I had the same problem when I had a "nan" in the data frame.

**Comment by slochower** [comment]
Ah, yes. I think it might be related to `nan` in the dataframe. I think this is the relevant snippet:

![image](https://user-images.githubusercontent.com/5393995/113314512-42ae6180-92c1-11eb-95a5-cca72cbd70d9.png)

**Comment by cbouy** [comment]
I've just published a fix in v0.0.4 which should fix the NaN issue, can you try it ?

**Comment by slochower** [comment]
Perfect.
